packages/type-utils/tests/TypeOrValueSpecifier.test.ts

```diff
@@ -202,6 +202,11 @@ describe('TypeOrValueSpecifier', () => {
         'type Foo = {prop: string}; type Test = Foo;',
         { from: 'file', name: 'Foo', path: 'tests/fixtures/file.ts' },
       ],
+      [
+        'type Foo = Promise<number> & {hey?: string}; let foo: Foo = Promise.resolve(5); type Test = typeof foo;',
+        // type.getSymbol() doesn't work here, .aliasSymbol does
```

Comment:
Nit: this feels pretty disconnected / out-of-context to place in the test. If there's a deeper technical explanation that's helpful to mention, I'd say it'd belong in the implementation.

```suggestion
```

Reply:
agreed, removed.